### PR TITLE
Update UG to include session management

### PR DESCRIPTION
**Pull request by ldinghan**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -766,6 +770,66 @@ Format: `editgrade INDEX ra1/READING_ASSESSMENT_1 ra2/READING_ASSESSMENT_2 mt/MI
...
+
+Format: `updatesessionremark s/SESSION_NUMBER r/REMARK`
+
+> **Parameters:**
+> * SESSION_NUMBER The session number of the session
+> * REMARK The new remark to be updated for the session
+
+Examples:
+* `updatesessionremark s/2 r/Teach Essence of Recursion` updates the remark for session number 2 to "Teach Essence of Recursion"
+* `updatesessionremark s/4 r/Taught streams and metacircular evalutator` updates the remark for session 4 to be "Taught streams and metacircular evaluator"
```
Comment (wesho1107) [inline docs/UserGuide.md]: Standardise grammar "to" or "to be"

**Review by wesho1107 — COMMENTED** [review]
Other than a nit, LGTM!

**Review by raydenlim — COMMENTED** [review]
Minor standardsiing issue, otherwise LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (ldinghan) [reply docs/UserGuide.md]: Fixed it!

**Review by Originalidk — APPROVED** [review]
LTGM!

**Pull request merged**